packages/embarkjs/embarkjs/src/lib/blockchain.js

```diff
@@ -89,6 +89,13 @@ Blockchain.setProvider = function(providerName, options) {
   provider.init(options);
 };
 
+Blockchain.isAvailable = function () {
+  if (!this.blockchainConnector) {
+    return Promise.resolve(false);
+  }
+  return this.blockchainConnector.getNetworkId().then(_ => true);
```

Comment:
Can `getNetworkId()` possibly reject? If so, should there be a `.catch(_ => false)` following `.then(_ => true)`?